microsoft-edge/extensions-chromium/whats-new/released-features.md

```diff
@@ -35,7 +35,7 @@ The sidebar feature of Microsoft Edge extensions enables an extension to display
 
 Microsoft Partner Center only accepts new Manifest V3 Microsoft Edge extension submissions, with visibility set as 'Hidden' or 'Public'.  To share your feedback and comments, see [Update your extensions from Manifest V2 to V3](https://github.com/microsoft/MicrosoftEdge-Extensions/discussions/27) in **Discussions** in the MicrosoftEdge-Extensions repo.
 
-*Released July 2022*
+*Released July 2022*<!-- todo: add date for complete removal? -->
```

Comment:
are any changes needed here, such as additional date for complete removal that's been done?
@bhuvanapriyap 

* **Released features for Microsoft Edge Add-ons**
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/whats-new/released-features?branch=pr-en-us-3401#retirement-of-submitting-new-microsoft-edge-add-ons-with-manifest-v2
   * External preview: https://github.com/MicrosoftDocs/edge-developer/blob/bhuvanapriyap-patch-2/microsoft-edge/extensions-chromium/whats-new/released-features.md#retirement-of-submitting-new-microsoft-edge-add-ons-with-manifest-v2
   * Before/live: https://learn.microsoft.com/microsoft-edge/extensions-chromium/whats-new/released-features#retirement-of-submitting-new-microsoft-edge-add-ons-with-manifest-v2